src/coreclr/jit/emitwasm.cpp

```diff
@@ -91,6 +96,52 @@ bool emitter::emitInsIsStore(instruction ins)
     return false;
 }
 
+size_t emitter::emitEncodeLEB64(uint8_t* destination, const void* source, bool valueIsSigned)
+{
+    assert(source);
+
+    unsigned char b;
+    unsigned char * originalDestination = destination;
+    if (valueIsSigned) {
+        int64_t value = *((int64_t*)source);
+        int more = 1, signBit;
+
+        while (more) {
+            b = (unsigned char)(value & 0x7FL);
+            value >>= 7;
+
+            signBit = (b & 0x40u) != 0;
+            if (
+                ((value == 0) && !signBit) ||
+                ((value == -1) && signBit)
+            )
+                more = 0;
+            else
+                b |= 0x80;
+
+            if (originalDestination)
```

Comment:
```suggestion
            if (originalDestination != nullptr)
```

Reply:
Also braces even around single-statement blocks

Reply:
jitformat didn't point this out, I'll make the update, thanks